src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'
+    const [businessName, setBusinessName] = useState("")
+    const [locationName, setLocationName] = useState("")
     const [currency, setCurrency] = useState(currencies.USD)
 
     const currencyList = Object.keys(currencies).map(key => (<MenuItem value={key}>{key}</MenuItem>))
 
+    function handleSubmit(){
+        const options = {
+            method: "POST",
+            headers: {
+              Accept: "application/json",
+              "Content-Type": "application/json;charset=UTF-8",
+            },
+            body: JSON.stringify({
+                'name': businessName,
+                'location_name': locationName,
+                'currency': currency
+            })}
+            fetch(url, options)
+            .then((response) => {response.json();
+                console.log(response);})
+         
+        
+          
+    }
+
     return (
         <FormControl>
             <TextField  id="standard-basic" variant="standard" sx={{mb:2}}

```

Comment:
every input form filed has a events, the once we want to use is called "onChange" and we need to specify a name for each form field, this name has to be exactly as defined starting on line 8 and we need to define the "handleChange" function

`<TextField onChange={handleChange} name="businessName" >`